containers/trtl/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Dynamic Builds
+ARG BUILDER_IMAGE=golang:1.16-buster
+ARG FINAL_IMAGE=debian:buster-slim
+
+# Build stage
+FROM ${BUILDER_IMAGE} as builder
+
+# Ensure ca-certificates are up to date
+RUN update-ca-certificates
+
+WORKDIR $GOPATH/src/github.com/trisacrypto/directory
+
+# Use modules for dependencies
+COPY go.mod .
+COPY go.sum .
+
+ENV CGO_ENABLED=0
+ENV GO111MODULE=on
+RUN go mod download
+RUN go mod verify
+
+# Copy only what is needed (ignore web and fixtures)
+COPY cmd ./cmd
+COPY pkg ./pkg
+COPY proto ./proto
+
+# Run tests to avoid building a broken container
+RUN go test -timeout 30s -v ./...
+
+# Build binaries
+RUN go build -v -o /go/bin/trtl ./cmd/trtl
+RUN go build -v -o /go/bin/gdsutil ./cmd/gdsutil
```

Comment:
Is gdsutil a strict requirement for using trtl? Can you remind me of what purpose gdsutil serves in regards to trtl?